### DISCUSSION TITLE
Build picker / simulator question

### DISCUSSION THREAD

**Discussion by tylerljames** [kkharji/xbase#219] [Q&A] [answered]
Hi, 

As a disclaimer, I am new to neovim and have been working in neovim for a few weeks now without problems. That being said I have been trying to configure xbase and seem to be up and running except for the pickers.

I am getting these errors when attempting to use any of the pickers, any help would be greatly appreciated.

![CleanShot 2023-05-21 at 12 07 38](https://github.com/kkharji/xbase/assets/86199305/6c72c00c-6efe-47c6-a450-c86ed571a251)

**Comment by kkharji** [comment, marked as answer]
Hello @tylergeist . Yes this was a bug. It is now fixed, please pull the latest changes

**Reply by tylerljames** [reply]
@kkharji , works perfectly now! Thank you for your quick work.
